examples/with-realm-web/pages/index.js

```diff
@@ -0,0 +1,95 @@
+import useSWR from 'swr'
+import { generateAuthHeader, REALM_GRAPHQL_ENDPOINT } from '../lib/RealmClient'
+
+const FIND_MOVIES = `
+  query FindMovies{
+    movies(query: { year: 2014, rated: "PG" } ) {
+      title
+      year
+      runtime
+    }
+  }
+`
+
+const fetcher = async (query) =>
+  fetch(REALM_GRAPHQL_ENDPOINT, {
+    method: 'POST',
+    headers: await generateAuthHeader(),
+    body: JSON.stringify({ query }),
+  }).then((res) => res.json())
+
+const IndexPage = () => {
+  const { data } = useSWR(FIND_MOVIES, fetcher)
+
+  if (data && data.error) {
+    console.error(data.error)
+    return <p>An error occurred: ${data.error}</p>
+  }
+  const movies = data ? data.data.movies : null
+
+  return (
+    <>
+      <div className="App">
+        <h1>"PG" Rated Movies - 2014</h1>
+
+        {data ? (
+          !movies && <div className="status">No movies Found</div>
+        ) : (
+          <div className="status"> Fetching data...</div>
+        )}
+
+        {movies && (
+          <table>
+            <thead>
+              <tr>
+                <th>#</th>
+                <th>Title</th>
+                <th>Runtime</th>
+              </tr>
+            </thead>
+            <tbody>
+              {data &&
+                movies.map((movie, index) => {
```

Comment:
You can remove the check for `data` here

Reply:
Missed that one out! Thanks, updated